src/main/java/com/unb/tracker/TrackerController.java

```diff
@@ -21,12 +17,23 @@ public String index(ModelMap map) {
     }
 
     @Autowired
+    private CourseRepository courseRepository;
+    @RequestMapping(value="/add-course", method = RequestMethod.GET, produces = {MediaType.TEXT_HTML_VALUE})
```

Comment:
We should probably change the request method to post or put and set the url to just "course"